File: mvvm-java/data/src/main/java/com/wasisto/githubuserfinder/android/data/searchhistory/SearchHistoryContract.java
package com.wasisto.githubuserfinder.android.data.searchhistory;

import android.provider.BaseColumns;

interface SearchHistoryContract {

    class SearchHistoryEntry implements BaseColumns {

        static final String TABLE_NAME = "search_history";

        static final String COLUMN_NAME_SEARCH_QUERY = "search_query";
    }
}

File: mvp-kotlin/data/src/main/java/com/wasisto/githubuserfinder/android/data/searchhistory/SearchHistoryContract.kt
package com.wasisto.githubuserfinder.android.data.searchhistory

import android.provider.BaseColumns

internal class SearchHistoryContract {

    object SearchHistoryEntry : BaseColumns {

        const val TABLE_NAME = "search_history"

        const val COLUMN_NAME_SEARCH_QUERY = "search_query"
    }
}